diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -61,8 +61,12 @@ DiffDriveController::init(const std::string & controller_name)
   try {
     auto node = get_node();
     // with the lifecycle node being initialized, we can declare parameters
-    node->declare_parameter<std::vector<std::string>>("left_wheel_names", {});
-    node->declare_parameter<std::vector<std::string>>("right_wheel_names", {});
+    node->declare_parameter<std::vector<std::string>>(
+      "left_wheel_names",
+      std::vector<std::string>());
```

Comment:
Propose to reduce the number of lines...

```suggestion
    node->declare_parameter<std::vector<std::string>>(
      "left_wheel_names", std::vector<std::string>());
```

Reply:
Ah, I simply ran `ament_uncrustify` and was under the impression there was only one valid outcome of that tool :)